src/main/handlers/helpers/saveUtils.ts

```diff
@@ -54,19 +55,15 @@ export const confirmSave: (
     throw new Error('Main window not defined');
   }
 
-  const cancelled = 2;
-
-  const confirmSaveDialogResponse = await dialog.showMessageBox(mainWindow, {
-    message: `Do you want to save the changes you have made to ${proposedFileName}?`,
-    detail: "Your changes will be lost if you don't save them",
-    buttons: ['Save', "Don't save", 'Cancel'],
-    defaultId: 0,
-  });
+  const confirmSaveDialogResponse = saveChangesDialog(
+    mainWindow,
+    proposedFileName
+  );
 
   // if cancel button is selected
-  if (confirmSaveDialogResponse.response === cancelled) {
+  if (confirmSaveDialogResponse === SAVE_CANCELLED) {
     throw new Error('Dialog cancelled');
```

Comment:
I know you didn't write this but I think throwing an error on dialog cancel isn't the right behaviour - can we just change this to console.log that it was cancelled